elasticsearch/README.md

```diff
@@ -101,7 +104,7 @@ make
 
 A cluster with X-Pack security enabled
 
-* Generate SSL certificates following the [official docs]( https://www.elastic.co/guide/en/elasticsearch/reference/6.3/configuring-tls.html#node-certificates)
+* Generate SSL certificates following the [official docs]( https://www.elastic.co/guide/en/elasticsearch/reference/6.4/configuring-tls.html#node-certificates)
```

Comment:
Why 6.4? TBH I have no idea why I linked to 6.3 but the 6.7 link seems to be the same too. Keeping them all the same will make it easier to do `s/6.7/6.8/g` on the file.

Reply:
6.4 appeared to be the last version that referenced x-pack for the linked documentation
> elasticsearch-certutil has been included with x-Pack

I thought this section was explicitly for x-pack security, but it is not, also this does not make sense since x-pack was opened started with 6.3 anyway.

I've updated to 6.7 😄